src/streamingCalls/streaming.ts

```diff
@@ -113,12 +121,29 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
    * @param {Stream} stream - The API request stream.
    */
   forwardEvents(stream: Stream) {
+    // Avoid "response" events emit twice.
+    let responseHasSent = false;
```

Comment:
I understand it works because of how closures work, but I feel it will be a little bit more readable if it's a private field in the class, not a variable in the function.

Reply:
sg